Slurm job started at 15/06/2023_19:20:27

NNODES = 6
SIMUNODES = 4
WORKERNODES = 2
NPROC = 128
NWORKER = 60
........['tcp://10.7.0.152:32860', 'tcp://10.7.0.152:33038', 'tcp://10.7.0.152:33402', 'tcp://10.7.0.152:34154', 'tcp://10.7.0.152:35417', 'tcp://10.7.0.152:35651', 'tcp://10.7.0.152:35720', 'tcp://10.7.0.152:36121', 'tcp://10.7.0.152:36288', 'tcp://10.7.0.152:36474', 'tcp://10.7.0.152:36532', 'tcp://10.7.0.152:37390', 'tcp://10.7.0.152:37942', 'tcp://10.7.0.152:38277', 'tcp://10.7.0.152:38443', 'tcp://10.7.0.152:38945', 'tcp://10.7.0.152:39932', 'tcp://10.7.0.152:40490', 'tcp://10.7.0.152:40802', 'tcp://10.7.0.152:41936', 'tcp://10.7.0.152:42232', 'tcp://10.7.0.152:42765', 'tcp://10.7.0.152:43623', 'tcp://10.7.0.152:44564', 'tcp://10.7.0.152:44686', 'tcp://10.7.0.152:44969', 'tcp://10.7.0.152:45928', 'tcp://10.7.0.152:46395', 'tcp://10.7.0.152:46411', 'tcp://10.7.0.152:46448', 'tcp://10.7.0.94:33633', 'tcp://10.7.0.94:34105', 'tcp://10.7.0.94:34129', 'tcp://10.7.0.94:34602', 'tcp://10.7.0.94:34770', 'tcp://10.7.0.94:35158', 'tcp://10.7.0.94:35312', 'tcp://10.7.0.94:35944', 'tcp://10.7.0.94:36010', 'tcp://10.7.0.94:36262', 'tcp://10.7.0.94:36772', 'tcp://10.7.0.94:37277', 'tcp://10.7.0.94:38679', 'tcp://10.7.0.94:38784', 'tcp://10.7.0.94:38852', 'tcp://10.7.0.94:39302', 'tcp://10.7.0.94:40554', 'tcp://10.7.0.94:41503', 'tcp://10.7.0.94:41977', 'tcp://10.7.0.94:42542', 'tcp://10.7.0.94:42709', 'tcp://10.7.0.94:42825', 'tcp://10.7.0.94:44433', 'tcp://10.7.0.94:44454', 'tcp://10.7.0.94:44463', 'tcp://10.7.0.94:45219', 'tcp://10.7.0.94:45874', 'tcp://10.7.0.94:46147', 'tcp://10.7.0.94:46721', 'tcp://10.7.0.94:46728']
dask.array<concatenate, shape=(25, 2048, 8192), dtype=float64, chunksize=(1, 256, 512), chunktype=numpy.ndarray>
Generated contract {'global_t': [(0, 25, 1)]}
Contract has been signed
Original arrays deleted
TOTAL_SIM_TIME:      20.634627277000003 (avg: 0.825385091080000)
SIM_BLOCK_TIME:      17.016348441000005 (avg: 0.680653937640000)
SIM_WTH_BLOCK:       3.618278835999998 (avg: 0.144731153440000)
SUCCESS
EST_TIME:            62.73832
Done

functions called:
tornado.application - ERROR - Exception in callback <bound method Client._heartbeat of <Client: 'tcp://10.7.0.25:8786' processes=60 threads=2400, memory=9.38 TiB>>
Traceback (most recent call last):
  File "/gpfs/users/fernandezx/spack/opt/spack/linux-centos7-cascadelake/gcc-11.2.0/py-tornado-6.2-lemkpcoofzciypjtwijrfhbso5bboah2/lib/python3.10/site-packages/tornado/ioloop.py", line 921, in _run
    val = self.callback()
  File "/gpfs/users/fernandezx/spack/opt/spack/linux-centos7-cascadelake/gcc-11.2.0/py-distributed-deisa-tvck22hdjddjrkrb4clzsednvt2igzro/lib/python3.10/site-packages/distributed/client.py", line 1099, in _heartbeat
    self.scheduler_comm.send({"op": "heartbeat-client"})
  File "/gpfs/users/fernandezx/spack/opt/spack/linux-centos7-cascadelake/gcc-11.2.0/py-distributed-deisa-tvck22hdjddjrkrb4clzsednvt2igzro/lib/python3.10/site-packages/distributed/batched.py", line 136, in send
    raise CommClosedError(f"Comm {self.comm!r} already closed.")
distributed.comm.core.CommClosedError: Comm <TCP (closed) Client->Scheduler local=tcp://10.7.0.25:39072 remote=tcp://10.7.0.25:8786> already closed.
ANALYTICS_TIME:      59.27
TRANSFER_TIME:       17.25
DESERIALIZE_TIME:    91.39
COMPUTE_TIME:        10.92
